apisix/admin/routes.lua

```diff
@@ -285,7 +285,7 @@ function _M.patch(id, conf, sub_path, args)
         end
         utils.inject_timestamp(node_value, nil, true)
     else
-        node_value = core.table.merge(node_value, conf);
+        node_value = core.table.merge(node_value, conf)
```

Comment:
Can we check it in the linter?

Reply:
Here are some ';' I have found with simple regex:
```
./admin/ssl.lua:        node_value = core.table.merge(node_value, conf);
./admin/plugin_config.lua:        node_value = core.table.merge(node_value, conf);
./admin/services.lua:        node_value = core.table.merge(node_value, conf);
./admin/global_rules.lua:        node_value = core.table.merge(node_value, conf);
./admin/upstreams.lua:        new_value = core.table.merge(new_value, conf);
./plugins/limit-req.lua:        key, err, n_resolved = core.utils.resolve_var(conf_key, ctx.var);
./plugins/limit-count.lua:        key, err, n_resolved = core.utils.resolve_var(conf_key, ctx.var);
./balancer/chash.lua:        chash_key, err, n_resolved = core.utils.resolve_var(key, ctx.var);
./plugins/limit-conn/init.lua:        key, err, n_resolved = core.utils.resolve_var(conf_key, ctx.var);
```

Let's remove them too.

Reply:
> Can we check it in the linter?

Great idea, I created an issue #6604 that I will add validation for in the next PR to make lint.